.github/workflows/release-beta.yml

```
@@ -0,0 +1,82 @@
+            echo "发布 @nutui/nutui-react 失败"
+            exit 1
+          fi
+
+      - name: Run Releases @nutui/nutui-react-taro
+        if: env.ABORT != 'true'
+        run: |
+          cd ./release/taro || exit 1
+          if ! npm publish --dry-run --tag ${{ env.RELEASE_TAG }}; then
+            echo "发布 @nutui/nutui-react-taro 失败"
+            exit 1
+          fi
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)